faq.html

```diff
@@ -139,31 +137,17 @@ <h5><b>Q. Will the Internet and WiFi setup be good enough for a large crowd? Sho
         <div class="pycon-faq">
           <h5><b>Q. What are the workshop, Developer Sprints and conference timings?</b></h5>
           <ul class="faq-answer">
-              <li> - Workshop day (Oct 5 2018): 08:00 AM - 06:30 PM.</li>
-             <li> - Developers Sprints days (Oct 8 and Oct 9, 2018): 08:00 AM - 06:30 PM.</li>
-              <li>- Conference days (Oct 6 and Oct 7, 2018): 08:00 AM </li>
-              <li>- 5.30 PM.</li>
+              <li>- Workshop day (Oct 5 2018): 08:00 AM - 06:30 PM.</li>
+              <li>- Conference days (Oct 6 and Oct 7, 2018): 08:00 AM - 5.30 PM.</li>
+              <li>- Developers Sprints days (Oct 8 and Oct 9, 2018): 08:00 AM - 06:30 PM.</li>
           </ul>
         </div> 
         <div class="pycon-faq">
           <h5><b>Q. What all does the Conference ticket (Oct 6 and Oct 7, 2018) include? </b></h5>
           <ul class="faq-answer">
-                <li> - Full conference pass for two days.</li>
-                <li>- Breakfast, Lunch and Hi-Tea</li>
-                <li>- PyCon India T-shirt which will be distributed on Oct 6, 2018.</li>
-          </ul>
-        </div> 
-        <div class="pycon-faq">
-          <h5><b>Q. My talk is not selected for conference, can I opt for OpenSpace? </b></h5>
```

Comment:
I guess we can comment this and the next questions/segment out as we'll be needing these in future instead of removing it completely.

Reply:
Done.